supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -466,6 +466,17 @@ Do not use "BIOS" as a generic term for computer firmware. Instead, write "firmw
 
 *See also*:
 
+[[bsl]]
+==== image:images/yes.png[yes] BSL (noun)
+*Description*: In the OpenShift API for Data Protection, the Data Protection Application backs up cluster data in the _backup storage location (BSL)_.
```

Comment:
Do you have any guidance for whether "BSL" needs to be spelled out upon first use, or if it can be used as "BSL" any time?

Reply:
It should be spelled out the first time.

Reply:
@apinnick Can you update the entry to instruct that?

Also, should the title then be "backup storage location" instead of "BSL"? Or perhaps "backup storage location (BSL)"?

Reply:
I updated the entry title to "backup storage location".